packages/shared/src/hooks/useTimer.ts

```diff
@@ -26,6 +27,18 @@ export default function useTimer<T = unknown>(
     }
   }, 1000);
 
+  useEffect(() => {
```

Comment:
What's this for? Maybe add a comment somewhere

Reply:
Just to be sure it will only be triggered once.

Reply:
Done 🚢